sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -10,6 +11,8 @@ class ConfigurationSetting(Model):
     """A configuration value.
     Variables are only populated by the server, and will be ignored when
     sending a request.
+    :ivar value: The value of the configuration setting
+    :vartype value: str
```

Comment:
if you add value here, we may want to remove 

:param value:
:type value: str

below.